{"content": {"suites": {"0": {"status": {"total_pass": 5, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 1, "total_error": 0}, "tests": {"0": {"status": "PASS", "message": "", "test_name": "test_verify_a_table_is_present_or_not", "rerun": "0"}, "1": {"status": "PASS", "message": "", "test_name": "test_verify_the_number_of_rows", "rerun": "0"}, "2": {"status": "PASS", "message": "", "test_name": "test_check_if_exists_in_hr_employees", "rerun": "0"}, "3": {"status": "PASS", "message": "", "test_name": "test_check_if_not_exists_in_hr_employees", "rerun": "0"}, "4": {"status": "FAIL", "message": "   AssertionError: maximum value does not match expected value, got: [(30, )]\n   assert [(30, )] == [(30,)]\n     At index 0 diff: (30, ) != (30,)\n     Full diff:\n     - [(30,)]\n     + [(30, )]\n     ?      +\n", "test_name": "test_verify_the_maximum_value_in_hr_dependents_dependent_id", "rerun": "0"}, "5": {"status": "PASS", "message": "", "test_name": "test_verify_the_values_range_for_hr_dependents_dependent_id", "rerun": "0"}}, "suite_name": "Module4PyTest/TestPyTest.py"}}}, "date": "October 30, 2022", "start_time": 1667119578.0868173, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "5", "fail": "1", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "6"}